guides/common/modules/proc_synchronizing-suse-content.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Synchronizing_SUSE_Content_{context}"]
+= Synchronizing SUSE Content
+
+You can use {Project} to synchronize SUSE content to deploy, attach, and serve content to managed hosts.
+
+.Procedure
+. Navigate to *Content > Products*, select the `SUSE Linux Enterprise Server 15 SP3 x86_64` product, and click *Sync Now* to synchronize the SUSE repositories for SLES 15 SP3 to {Project}.
+. Navigate to *Content > Content Views* and click *Create Content View*.
+.. Create a content view called `SLES 15 SP3` comprising the SLES repositories created in the `SLES 15 SP3` product.
+.. Create a content view called `SLES 15 SP3 {Project} client` comprising the {Project} client repository created in the `SLES 15 SP3 {Project} client` product.
+.. Publish a new version of both xref:Promoting_a_Content_View_{context}[content views].
+. Navigate to *Content > Content Views* and click *Create Content View*.
+.. xref:Creating_a_Composite_Content_View_{context}[Create a composite content view] called `Composite SLES 15 SP3` comprising the previously published `SLES 15 SP3` content view, the `SLES 15 SP3 {Project} client` content view, and optionally further content views of your choice, for example a content view containing Puppet.
```

Comment:
As I commented before, this is inconsistent with the other guides, but I will leave it up to you. 
Create a composite content view called x comprising of y. For more information on creating a Composite Content View, see x .

Reply:
Moved the link as you suggested.